mozjs-sys/src/jsimpls.rs

```diff
@@ -302,6 +302,11 @@ impl JS::CallArgs {
             JS::MutableHandleValue::from_marked_location(self.argv_.offset(self.argc_ as isize))
         }
     }
+
+    #[inline]
+    pub unsafe fn is_constructing(&self) -> bool {
```

Comment:
Let's use an unsafe block instead of marking the method as unsafe.

Reply:
Thanks for the review. I will fix it.

Reply:
Fixed. Could you take a look again when you have time, please?